TEAMS.md

```diff
@@ -0,0 +1,31 @@
+# OpenGitOps teams
+
+## What and why
+
+Community interest in GitOps, the GitOps Working Group, and OpenGitOps project has continued to increase over the past several years. The WG is now large enough – and there are enough work products the group is committed to – that the project work can not all be organized within the all-group twice-monthly meeting. Often these areas of focus will be worked on by only a subset of the WG. Subgroups – or teams – have been a very functional solution to helping this work happen in a focused way while keeping the group on track. Every twice-monthly all-group meeting starts with a report by each of the teams, including new members. This could include you!
+
+## Process
+
+To join an existing team - [listed below](#list-of-teams) - it's a good idea to start by joining the twice monthly GitOps WG meeting and introduce yourself. You may skip this if you are unable to join those meetings. In either case, for transparency, open a PR adding yourself to the list below in [this file](./TEAMS.md), and in the PR description explain which team(s) you want to volunteer to join and why. All legitimate volunteers who are able to continue contributing are welcome to join any team. Once the PR is approved, the user should be added to the appropriate GitHub team in the [open-gitops](https://github.com/open-gitops) GitHub org as a last step before the PR is merged. New team members should be added to the the next [GitOps WG twice-monthly meeting agenda](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit) so they can be welcomed by the group. 🦄
```

Comment:
```suggestion
To join an existing team - [listed below](#list-of-teams) - it's a good idea to start by joining the twice monthly [GitOps WG meeting](https://zoom.us/j/93779536510?pwd=TEFzbGRzREI3MVlkZmluemVkMEhHdz09) (1st and 3rd Wednesday - 19:00 GMT) and introduce yourself. You may skip this if you are unable to join those meetings. In either case, for transparency, open a PR adding yourself to the list below in [this file](./TEAMS.md), and in the PR description explain which team(s) you want to volunteer to join and why. All legitimate volunteers who are able to continue contributing are welcome to join any team. Once the PR is approved, the user should be added to the appropriate GitHub team in the [open-gitops](https://github.com/open-gitops) GitHub org as a last step before the PR is merged. New team members should be added to the the next [GitOps WG twice-monthly meeting agenda](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit) so they can be welcomed by the group. 🦄
```

Maybe adding more information about when the meeting happen? And also adding the link so folks can bookmark it for future attendance.